evap/results/templates/results_evaluation_detail.html

```diff
@@ -72,6 +72,13 @@ <h3>{{ evaluation.full_name }} ({{ evaluation.course.semester.name }})</h3>
     <div class="card card-outline-primary mb-3">
         <div class="card-header d-flex">
             <div class="mr-auto">{% trans 'Overview' %}</div>
+            {% if view == "export" or view == "full" %}
+                <div class="ml-auto mr-3 my-auto">
```

Comment:
please pay attention that the button is currently changing the card-header's size which it shouldn't. also, please include `d-print-none`, as the button should not be visible when printing the results page.

Reply:
the button is now placed too close to the grade document download button. can you please add some spacing?